Formula/minio.rb

```diff
@@ -21,12 +15,21 @@ def install
     clipath = buildpath/"src/github.com/minio/minio"
     clipath.install Dir["*"]
 
-    cd buildpath/"src/github.com/minio/minio/" do
-      system "go", "build", "-o", bin/"minio", "-v"
+    cd clipath do
+      if build.head?
+        system "go", "build", "-o", buildpath/"minio"
+      else
+        release = "RELEASE.2016-07-13T21-46-05Z"
+        commit = "3f27734c22212f224037a223439a425e6d2b653a"
```

Comment:
having some issues retrieving the current tag from the buildsystem. Trying to catch the output of `"git", "tag", "--points-at", "HEAD"` and save it into variable, but pipe_output / shell_output are not available. Any suggestions?